src/api/acl.py

```
@@ -0,0 +1,40 @@
+from fastapi import Depends
+from fastapi.responses import JSONResponse
+
+from config.settings import get_index_name
+from dependencies import get_current_user, get_session_manager
+from session_manager import User
+
+
+async def get_document_acl(
+    filename: str,
+    session_manager=Depends(get_session_manager),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)